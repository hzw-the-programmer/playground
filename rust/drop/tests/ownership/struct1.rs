fn main() {
    let f1 = Object { id: 1 };
    let f2 = Object { id: 2 };
    let s = S { f1, f2 };

    let S { f1, f2 } = s;
    println!("{:?} {:?}", f1, f2);
    println!("{:?}", s);
    println!("finish");
}