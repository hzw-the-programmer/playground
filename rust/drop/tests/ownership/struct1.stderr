error[E0422]: cannot find struct, variant or union type `Object` in this scope
 --> tests/ownership/struct1.rs:2:14
  |
2 |     let f1 = Object { id: 1 };
  |              ^^^^^^ not found in this scope
  |
help: consider importing this struct
  |
1 | use drop::Object;
  |

error[E0422]: cannot find struct, variant or union type `Object` in this scope
 --> tests/ownership/struct1.rs:3:14
  |
3 |     let f2 = Object { id: 2 };
  |              ^^^^^^ not found in this scope
  |
help: consider importing this struct
  |
1 | use drop::Object;
  |

error[E0422]: cannot find struct, variant or union type `S` in this scope
 --> tests/ownership/struct1.rs:4:13
  |
4 |     let s = S { f1, f2 };
  |             ^ not found in this scope
  |
help: consider importing this struct
  |
1 | use drop::S;
  |

error[E0422]: cannot find struct, variant or union type `S` in this scope
 --> tests/ownership/struct1.rs:6:9
  |
6 |     let S { f1, f2 } = s;
  |         ^ not found in this scope
  |
help: consider importing this struct
  |
1 | use drop::S;
  |
